lightquakevisualizer/lightQuakeVisualizer.py

```diff
@@ -737,15 +744,17 @@ def get_snapshot_fname(args, fname, itime, time_value):
             basename = args.output_prefix.replace("%d", f"_{itime}")
             if "{t" in basename:
                 basename = basename.format(t=float(time_value))
-
         else:
             mod_prefix = os.path.splitext(fname)[0].replace("/", "_")
             svar = args.variables.replace(";", "_")
             view_name, view_ext = os.path.splitext(os.path.basename(args.view))
             is_pvcc = view_ext == ".pvcc"
             spvcc = f"_{view_name}_" if is_pvcc else ""
             basename = f"{mod_prefix}{spvcc}{svar}_{itime}"
-        return f"output/{basename}.png"
+        if args.output_dir:
```

Comment:
just 
return f"{args.output_dir}/{basename}.png"
args.output_dir is always defined becuase of the default value.

but best is 
return os.path.join(args.output_dir, f"{basename}.png")

then you can use empty output_dir.